Saint-Nec-Engine/Saint-Nec-Engine-Lib/src/physics/BoxCollider.hpp

```diff
@@ -8,27 +8,100 @@ namespace sne
 
     namespace saintNecPhysics
     {
+        /**
+         * @class BoxCollider
+         * @brief BoxCollider physics component
+         */
         class BoxCollider : public sne::Component
         {
         private:
-            Vector3 _points[8]; // TO UPDATE
+            Vector3 _points[8];         // TO UPDATE
             Vector3 _rotation{0, 0, 0}; // Radian
             Vector3 _center;
 
         public:
-            // TO UPDATE
+            /**
+             * @brief Construct a new Box Collider object
+             * 
+             * @param center 
+             * @param width 
+             * @param depth 
```

Comment:
Might be a godd idea to associate those 3 parameters with an axis (x y or z)

Reply:
That's a good idea. Indeed, idk how but I really thought about like how am I supposed to describe it the best way, and thought about it geometrically, in my head. Sad to see I wasn't smart enough to use those basic letters :smiling_face_with_tear: